src/pages/Signup/index.jsx

```diff
@@ -112,16 +113,22 @@ const SignupScreen = () => {
     }
 
     if (submitable) {
-      const usernameDoc = db.doc(`usernames/${username}`);
+      const usernameDoc = db.collection(`users`);
       const batch = db.batch();
```

Comment:
we are not using batch right? so we can remove it

Reply:
yeah that's fine.